client/coniksclient/internal/cmd/lookup.go

```diff
@@ -61,6 +67,10 @@ var lookupCmd = &cobra.Command{
 					fmt.Println("Cannot get the key from the response, error: " + err.Error())
 				} else {
 					fmt.Println("Success! Key bound to name is: [" + string(key) + "]")
+					bindings[name] = string(key)
+					if err := writeKeys(bindings, conf.KeyStoragePath); err != nil {
+						fmt.Println("Cannot save the key to the file. Error: " + err.Error())
+					}
```

Comment:
Hmm, I see. I guess for the test client that's fine. For an actual client, I think the error should probably be treated separately since it reflects a consistency violation error, which is a likely indication of misbehavior, as opposed to, say, a signature verification error, which might be a misconfiguration error. Either way, we probably want to start thinking about how to report these different errors to the user.